docs/metrics/tp-fp-fn-tn.md

```diff
@@ -16,18 +16,25 @@ many other metrics, including accuracy, precision, and recall.
 | <nobr>False Negative</nobr> | FN | An instance for which predicted value is negative but actual value is positive |
 | True Negative | TN | An instance for which both predicted and actual values are negative |
 
-To compute these metrics, each prediction is compared to a ground truth and categorized into one of the four groups.
+To compute these metrics, each inference is compared to a ground truth and categorized into one of the four groups.
 Let’s say we’re building a dog classifier that predicts whether an image has a dog or not:
 
-![A dog classifier example](../assets/images/metrics-tpfpfntn-dog-classifier.png)
+<center>
+
+|                              | Positive Inference (Dog) | Negative Inference (No Dog) |
+|                              | :----------------------: | :-------------------------: |
+| **Positive Sample (Dog)**    | True Positive (TP)       | False Negative (FN)         |
+| **Negative Sample (No Dog)** | False Positive (FP)      | True Negative (TN)          |
```

Comment:
I would replace `Sample` here with `Ground Truth` to ensure we're using consistent terminology